### PR TITLE
ci: add testTimeout of 10 seconds

### PR DESCRIPTION
**Pull request by maschad**
master is currently [failing](https://github.com/FuelLabs/fuels-ts/actions/runs/8839609115/job/24274967664#step:10:712)

### CODE REVIEW

File: packages/fuel-gauge/package.json [hunk 1]
```diff
@@ -16,8 +16,9 @@
   },
   "devDependencies": {
     "@fuel-ts/account": "workspace:*",
+    "@fuel-ts/errors": "workspace:*",
     "@fuel-ts/forc": "workspace:*",
     "@fuel-ts/utils": "workspace:*",
-    "@fuel-ts/errors": "workspace:*"
+    "delay": "^6.0.0"
```
Comment (arboleya) [inline packages/fuel-gauge/package.json:22]: We could probably move:
 - [packages/account/src/providers/utils/sleep.ts](https://github.com/FuelLabs/fuels-ts/blob/9c3c094d810b8400e8c0b9ac2830f75ca2ca70b5/packages/account/src/providers/utils/sleep.ts)

To:
 - [packages/utils/src/utils](https://github.com/FuelLabs/fuels-ts/tree/9c3c094d810b8400e8c0b9ac2830f75ca2ca70b5/packages/utils/src/utils)

Then, import and use it from there instead `delay`.

```suggestion
```

Comment (arboleya) [reply packages/fuel-gauge/package.json:22]: Example:

https://github.com/FuelLabs/fuels-ts/blob/9c3c094d810b8400e8c0b9ac2830f75ca2ca70b5/packages/account/src/providers/utils/auto-retry-fetch.ts#L82

**Comment by maschad** [comment]
@arboleya I'm wondering if this is even the root cause, I'm noticing that a bash script is reporting that [forc](https://github.com/FuelLabs/fuels-ts/actions/runs/8840613546/job/24276271224?pr=2173#step:10:702) and [fuel-core](https://github.com/FuelLabs/fuels-ts/actions/runs/8840613546/job/24276271224?pr=2173#step:10:703) are not found, could that be related? @Torres-ssf

**Comment by maschad** [comment]
Looks like https://github.com/FuelLabs/fuels-ts/pull/2175 has addressed this issue

**Pull request closed without merging**